pkgs/development/python-modules/babeltrace/default.nix

```diff
@@ -0,0 +1,14 @@
+{ toPythonModule
+, python
+, babeltrace
+, swig2
+}:
+
+toPythonModule (babeltrace.overrideAttrs ({ nativeBuildInputs ? [ ], configureFlags ? [ ], ... }: {
+  nativeBuildInputs = nativeBuildInputs ++ [ swig2 ];
+
+  configureFlags = configureFlags ++ [ "--enable-python-bindings" ];
```

Comment:
Why not put this logic in an `enablePython` flag?

Reply:
See above.